src/admin/content-block/content-block.types.ts

```diff
@@ -127,3 +129,10 @@ export interface IntroBlockComponentState {
 	text: string;
 	align: AlignOptions;
 }
+
+export interface CTAsBlockComponentState {
+	heading: string;
+	headingType: HeadingLevelOptions;
+	content: string | string[];
+	// button: Partial<ButtonsBlockComponentState>;
```

Comment:
waarom gecomment?

Reply:
Content Block is er nog niet op voorbereid.